added goerli and sepolia inverse colors

NidhiKJha opened the pull request:
This PR is to update `COLORS.SEPOLIA_INVERSE` and `COLORS.GOERLI_INVERSE `

Fixes: #16787

NidhiKJha committed:
added goerli and sepolia inverse colors

georgewrmarshall commented on ui/components/ui/box/box.stories.js:
```diff
@@ -346,6 +346,12 @@ export const Color = (args) => {
       <Box {...args} padding={3} color={COLORS.INHERIT} {...args}>
         COLORS.INHERIT
       </Box>
+      <Box {...args} padding={3} color={COLORS.SEPOLIA_INVERSE} {...args}>
+        COLORS.SEPOLIA_INVERSE
+      </Box>
+      <Box {...args} padding={3} color={COLORS.GOERLI_INVERSE} {...args}>
+        COLORS.GOERLI_INVERSE
+      </Box>
```
Can you add the network defaults as background colors as well?

```suggestion
      <Box {...args} padding={3} backgroundColor={COLORS.SEPOLIA} color={COLORS.SEPOLIA_INVERSE} {...args}>
        COLORS.SEPOLIA_INVERSE
      </Box>
      <Box {...args} padding={3} backgroundColor={COLORS.GOERLI} color={COLORS.GOERLI_INVERSE} {...args}>
        COLORS.GOERLI_INVERSE
      </Box>
```

georgewrmarshall reviewed (commented):
Nice! Can you also add it to the TEXT_COLORS and ICON_COLORS consts respectively they should have the same coverage as PRIMARY_DEFAULT/PRIMARY_INVERSE

Could add to the controls for color
<img width="970" alt="Screenshot 2022-12-13 at 9 47 20 AM" src="https://user-images.githubusercontent.com/8112138/207407258-fe3257b3-025d-48fc-afa1-752e3ff0dd69.png">

Could you also add GOERLI and SEPOLIA to the Background Color story 

<img width="1435" alt="Screenshot 2022-12-13 at 9 42 17 AM" src="https://user-images.githubusercontent.com/8112138/207406828-9ad80219-3cc2-4e2f-b30f-ca11321cd8cd.png">

NidhiKJha committed:
updated story

georgewrmarshall commented on ui/components/ui/box/box.stories.js:
```diff
@@ -346,6 +346,24 @@ export const Color = (args) => {
         COLORS.INHERIT
       </Box>
+      <Box
+        {...args}
+        padding={3}
+        backgroundColor={COLORS.SEPOLIA}
+        color={COLORS.SEPOLIA_INVERSE}
+        {...args}
+      >
+        COLORS.SEPOLIA_INVERSE
+      </Box>
+      <Box
+        {...args}
+        padding={3}
+        backgroundColor={COLORS.GOERLI}
+        color={COLORS.GOERLI_INVERSE}
+        {...args}
```
Ahh looks like I was spreading `{...args}` twice here 🤦 could you please remove the duplication for the entire story/file 🙏

georgewrmarshall reviewed (changes requested):
Hmm just realized that the inverse colors for sepolia and goerli should be white. Doesn't look like the class name is being attached and even when I added `box--text-color-sepolia-inverse` it didn't seem to work so not sure if the class name is being generated either?

I think you also need to add the inverse options to `ui/css/design-system/colors.scss`

<img width="1440" alt="Screenshot 2022-12-14 at 9 09 28 AM" src="https://user-images.githubusercontent.com/8112138/207662181-c824c064-91c4-43cc-b758-755a033a337d.png">

NidhiKJha committed:
removed duplication of args

NidhiKJha committed:
updated colors in root

NidhiKJha commented:
> Hmm just realized that the inverse colors for sepolia and goerli should be white. Doesn't look like the class name is being attached and even when I added `box--text-color-sepolia-inverse` it didn't seem to work so not sure if the class name is being generated either?
> 
> I think you also need to add the inverse options to `ui/css/design-system/colors.scss`
> 
> <img alt="Screenshot 2022-12-14 at 9 09 28 AM" width="1440" src="https://user-images.githubusercontent.com/8112138/207662181-c824c064-91c4-43cc-b758-755a033a337d.png">

Nice Catch! Updated it here 

<img width="1496" alt="Screenshot 2022-12-14 at 10 54 44 PM" src="https://user-images.githubusercontent.com/39872794/207664672-c5b2378d-a914-4183-9add-5893cb379eda.png">

NidhiKJha committed:
Merge branch 'develop' into fix-16787

georgewrmarshall reviewed (approved):
Nicely done 🤙 💯
